test/libsolidity/semanticTests/externalContracts/deposit_contract.sol

```diff
@@ -178,35 +178,35 @@ contract DepositContract is IDepositContract, ERC165 {
 // compileViaYul: also
 // ----
 // constructor()
-// gas irOptimized: 1657475
+// gas irOptimized: 1558013
 // gas legacy: 2580394
-// gas legacyOptimized: 1775403
+// gas legacyOptimized: 1776483
```

Comment:
Weird, why did this change?

Reply:
Good question, I hadn't noticed...

Reply:
Just pushed it with the old value - if the tests pass now as well, this confirms that there's something broken with those...